### PR TITLE
added missing requirements to allow easy setup for developers

### PR DESCRIPTION
**Pull request by ratara**
Open idea:
- I would suggest fixed versions for the requirements to maintain a stable version and to prevent unforseen breaking changes. But we have to maintain that versions and i am not sure, if there is somebody willing to do so.

### CODE REVIEW

**Comment by ratara** [comment]
Well, I guess these unit test fails are meant, if we are speaking of unstable? - I obviously did not change any behaviour.

**Comment by oroulet** [comment]
your branch is out of date . Rebase on master. I think the last commit os the one fixing that issue

File: dev_requirements.txt [hunk 1]
```diff
@@ -4,3 +4,9 @@ coverage
 pytest-cov
 pytest-mock
 asynctest
+python-dateutil
+pytz
+aiofiles
+cryptography
+sortedcontainers
+aiosqlite
```
Comment (oroulet) [inline dev_requirements.txt:12]: here you are repeated dependencies which are already in requirement.py (thorugh the use of a dot, so the dependencies are fetched from setup.py)

Comment (oroulet) [reply dev_requirements.txt:12]: so not, we should not do that

**Pull request closed without merging**